### PR TITLE
Fix documentation issues

### LINKED ISSUE #72: Documentation issues
- https://pypi.org/project/anticipy/#description - Page formatting messed up. We were previously reusing the github README.md file, but this is no longer possible because badges break pypi formatting. We will add a new file, pypi_description.md
- https://anticipy.readthedocs.io/en/latest/tutorial.html - several typos

### PR DESCRIPTION
**Pull request by capelastegui**
close #72 #66

### CODE REVIEW

File: anticipy/forecast.py [hunk 1]
```diff
@@ -655,7 +656,7 @@ def extrapolate_model(
         weekly samples, or 'D' for daily samples.
         (untested) Any date unit or time unit accepted by numpy should also
         work, see
-        https://docs.scipy.org/doc/numpy-1.13.0/reference/arrays.datetime.html#arrays-dtypes-dateunits # noqa
+        https://docs.scipy.org/doc/numpy-1.13.0/reference/arrays.datetime.html#arrays-dtypes-dateunits #noqa
```
Comment (slenas) [inline anticipy/forecast.py]: Travis complains about this line. Probably it needs the space.

File: anticipy/forecast_models.py [hunk 2]
```diff
@@ -250,35 +251,51 @@ class ForecastModel:
...
+        # Get parameter initialisation function
+        f_init_params = model_x.f_init_params
+        # Get initial parameters
+        init_params = f_init_params(t_values, y_values)
+        # Get model fitting function
+        f_model = model_x.f_model
+        # Get model output
+        y = f_model(a_x, a_date, parameters)
 
     The following pre-generated models are available. They are available as attributes from this module:
```
Comment (slenas) [inline anticipy/forecast_models.py]: This line also needs fixing. Travis complains about being too long.

File: anticipy/forecast_plot.py [hunk 3]
```diff
@@ -329,12 +329,12 @@ def plot_forecast(df_fcast, output, path=None, width=None, height=None,
...
-        | - model (str) : ID for the forecast model
-        | - y (float) : Value of the time series in that sample
-        | - is_actuals (bool) : True for actuals samples, False for
-        |                       forecasted samples # noqa
+      |  Forecast Dataframe with the following columns:
+      |  - date (timestamp)
+      |  - model (str) : ID for the forecast model
+      |  - y (float) : Value of the time series in that sample
+      |  - is_actuals (bool) : True for actuals samples, False for forecast
+      |
```
Comment (slenas) [inline anticipy/forecast_plot.py]: what's the purpose of this blank line?

File: anticipy/model_utils.py [hunk 4]
```diff
@@ -128,7 +128,7 @@ def get_s_x_extrapolate(
     :param freq: Time unit between samples. Supported units are 'W' for weekly
         samples, or 'D' for daily samples. (untested) Any date unit or time
         unit accepted by numpy should also work, see
-        https://docs.scipy.org/doc/numpy-1.13.0/reference/arrays.datetime.html#arrays-dtypes-dateunits  # noqa
+        https://docs.scipy.org/doc/numpy-1.13.0/reference/arrays.datetime.html#arrays-dtypes-dateunits #noqa
```
Comment (slenas) [inline anticipy/model_utils.py]: needs the space apparently

File: docs/source/tutorial.rst [hunk 5]
```diff
@@ -81,7 +81,7 @@ An input dataframe should meet the following constraints:
   - Minimum required samples depends on number of parameters in the chosen models. run_forecast() will fail to fit if
     n_samples < n_parameters+2
   - y column may include null values.
-  - Multiple values per sample may be included, e.g when the same metric is observed by multiple sources. In that case,
+  - Multiple values per sample may be included, e.g. when the same metric is observed by multiple sources. In that case,
     it is possible to assign weights to each individual sample so that one source will be given higher priority. To do
```
Comment (slenas) [inline docs/source/tutorial.rst]: Does it get to the next line here?

File: anticipy/forecast_plot.py (hunk 3 above)
Comment (capelastegui) [reply anticipy/forecast_plot.py]: Bug - I'll remove it.

**Comment by capelastegui** [comment]
I have fixed the problem with 3 long lines that made pep8 complain. I had started by removing all # noqa annotations, but these lines had hyperlinks that couldn't be split, so I added back the noqa - with a typo. This is again working as intended. 

BTW, there is a known problem with # noqa annotations in docs and sphinx: I haven't found any way to make sphinx aware of pep8 annotations, so the text 'noqa' is actually showing up in our docs. The best solution is to fix the code problems so that the annotations are no longer necessary. Unfortunately, we can't really do that with hyperlinks that are more than 80 characters long. Any suggestions are welcome, but for now I think we'll merge with the slightly bugged docs.

File: docs/source/tutorial.rst (hunk 5 above)
Comment (slenas) [reply docs/source/tutorial.rst]: if it's a continuing line no problem then. It seemed that there were more spaces there so the line was breaking without an actual reason...

**Comment by capelastegui** [comment]
One more thing... after committing fixes, I saw that the comment UI in github has an option to propose quick code changes from the web. We could try that out the next time we have a bunch of single-character errors.

**Pull request merged**